README.md

```diff
@@ -23,4 +23,4 @@ Implement the `App` and `Tabs` components to render and switch between given `ta
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_tabs/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://VovaKlimyuk.github.io/react_tabs/) and add it to the PR description.
```

Comment:
https://user-images.githubusercontent.com/38065883/225937737-5162f794-3b36-436f-b068-11cdf39f146e.mov

There should be the same tab after reload